docs/docs/setup.md

```diff
@@ -40,15 +40,17 @@ Arrow Meta IDE Plugin is compatible with the same versions as [Kotlin Plugin for
 
 When using the Arrow Meta Gradle Plugin from Intellij IDEA, there is a Gradle task available to install the Arrow Meta IDE Plugin.
 
-This message will appear with every Gradle task execution if Arrow Meta IDE Plugin is not installed, for instance, when running a `clean` task:
+This message will appear with every Gradle task execution if Arrow Meta IDE Plugin isn't installed, for instance, when running a `clean` task:
 
 ```
 > Configure project :
 Arrow Meta IDE Plugin is not installed!
-Run 'install-idea-plugin' Gradle task under 'Arrow Meta' group to install it.
+Run 'install-idea-plugin' Gradle task under 'Arrow Meta' group to install it (choose just one project when multi-project)
+Receive update notifications when adding this custom repository: https://meta.arrow-kt.io/idea-plugin/latest-<version-type>/updatePlugins.xml
+Guideline: https://www.jetbrains.com/help/idea/managing-plugins.html#repos
 ```
 
-Go to `Arrow Meta` group for Gradle tasks and run `install-idea-plugin`:
+Go to `Arrow Meta` group for Gradle tasks (choose just one project when multi-project) and run `install-idea-plugin`:
```

Comment:
Nit: same here.. instead of 'Arrow Meta' I would say 'Tasks -> arrow meta'

Reply:
Changed, thanks @danimontoya :raised_hands: